packages/network/src/config.ts

```diff
@@ -102,50 +280,23 @@ export const mainnetNetworks = createNetworkConfig(
 export const testnetNetworks = createNetworkConfig(
   (vars: { [key: string]: any }) => [
     {
-      name: 'rinkeby',
-      chainId: ChainId.RINKEBY,
+      ...networks[ChainId.RINKEBY],
       ensAddress: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
-      testnet: true,
       rpcUrl: urlClean(`${vars.baseRpcUrl}/rinkeby`),
       relayer: { url: urlClean(`${vars.baseRelayerUrl}/rinkeby`) },
       isDefaultChain: true
     },
     {
-      name: 'goerli',
-      chainId: ChainId.GOERLI,
+      ...networks[ChainId.GOERLI],
       ensAddress: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
-      testnet: true,
       rpcUrl: urlClean(`${vars.baseRpcUrl}/goerli`),
       relayer: { url: urlClean(`${vars.baseRelayerUrl}/goerli`) },
       isAuthChain: true
     }
-    // {
-    //   name: 'ropsten',
-    //   chainId: ChainId.ROPSTEN,
-    //   ensAddress: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
-    //   rpcUrl: '',
-    //   // relayer: null
-    // },
-    // {
-    //   name: 'kovan',
-    //   chainId: ChainId.KOVAN,
-    //   rpcUrl: '',
-    //   relayer: null
-    // },
-    // {
-    //   name: 'mumbai',
-    //   chainId: ChainId.POLYGON_MUMBAI,
-    //   rpcUrl: 'https://rpc-mumbai.matic.today',
-    //   relayer: null
-    // }
```

Comment:
Do you happen to know why Mumbai wasn't in the list of testnets? Should check with whoever commented it out in the first place maybe...

Reply:
It has always been commented out, decided to just remove the comment, we can add back in the full config if we want it to be a default test net in the future.